src/themes/rdc/app/item-page/simple/item-types/untyped-item/untyped-item.component.html

```diff
@@ -14,6 +14,7 @@
   </ds-themed-item-page-title-field>
   <ds-dso-edit-menu></ds-dso-edit-menu>
 </div>
+
```

Comment:
Let's try to keep the changes to files in the themes that are duplicate of core code to a minimal to simplify checking to synchronize on next update.